build.gradle

```diff
@@ -21,7 +21,7 @@ allprojects {
 
     version = '1.0'
     ext {
-        appName = "the-legend-of-ericc"
+        appName = "The Legend Of Eric"
```

Comment:
gradla też nie zmieniaj, to lokalny plik

Reply:
W sensie np. appName nie ma sensu zmieniać co każdy push, same dependencje sa oks

Reply:
co to znaczy zmieniać co każdy push?